maps/adhomai/code/machinery/biogenerator.dm

```diff
@@ -0,0 +1,38 @@
+/obj/machinery/biogenerator/adhomai/biogenerator
```

Comment:
/obj/machinery/biogenerator/adhomai is enough here.

Reply:
Fixed.